### PR TITLE
Create parent attendance display #130

### PR DESCRIPTION
**Pull request by willji0023**
### Administrative Info

Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Github Issue]-[3-4 word description separated by dashes]`.

### What issue(s) does this branch close?

#130 

### Changes

What changes did you make?

- Create route to return all classes for one student
- Added frontend for rhs module on parents profile page that updates when student is changed

### Testing

How did you confirm your changes worked?

- Tested route and frontend functionality locally by creating students enrolled in classes and adding them to a parent

### Confirmation of Change

Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

| Parent's Profile page |
| ----------------------- |
| image |

### CODE REVIEW

File: components/Profile/ParentProfile/StudentClasses.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+              <CalendarMonth />{" "}
+              <div className={styles.info}>
+                {RRule.fromString(Class.rrstring).toText().charAt(0).toUpperCase() +
+                  RRule.fromString(Class.rrstring).toText().slice(1) +
+                  ", start " +
+                  monthNames[RRule.fromString(Class.rrstring).options.dtstart.getMonth()] +
+                  " " +
+                  RRule.fromString(Class.rrstring).options.dtstart.getDate() +
+                  ", " +
+                  RRule.fromString(Class.rrstring).options.dtstart.getFullYear()}
```
Comment (Anshul-Birla) [inline components/Profile/ParentProfile/StudentClasses.tsx:95]: can we add a comment about what this is doing - its kinda hard to understand

File: components/Profile/ParentProfile/StudentClasses.tsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+const getFormattedTime: string = (time: string) => {
+  const hours24 = parseInt(time.substring(0, 2));
+  const hours = ((hours24 + 11) % 12) + 1;
+  const hoursText = hours < 10 ? "0" + hours.toString() : hours.toString();
+  const amPm = hours24 > 11 ? " pm" : " am";
+  const minutes = time.substring(3, 5);
+
+  return hoursText + ":" + minutes + amPm;
+};
```
Comment (Anshul-Birla) [inline components/Profile/ParentProfile/StudentClasses.tsx:29]: just wondering, is there no better way to do this? this forces us to keep the exact same formatting through the entire app. Maybe luxon would help here? B/c now we have to make sure the formatting is correct

**Review by Anshul-Birla — CHANGES_REQUESTED** [review]
Hey william, let some very minor comments on your PR. Everything else looks good to me. Also, unit tests are failing

File: components/Profile/ParentProfile/StudentClasses.tsx (hunk 2 above)
Comment (willji0023) [reply components/Profile/ParentProfile/StudentClasses.tsx]: Yes, I agree I think something like luxon is a cleaner option. Added in new iteration

**Review by Anshul-Birla — APPROVED** [review]
Everything is great! Enjoy your new team :)

**Pull request merged**